eks-anywhere-common/Testers/Pulumi/pulumi-tester-job.yaml

```diff
@@ -0,0 +1,169 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: pulumi-tester-configmap
```

Comment:
Consider naming this `pulumi-tester-scripts` or something to disambiguate from the other configmap. I don't know much about the other's purpose.